### DISCUSSION TITLE
Publishing Docker images

### DISCUSSION THREAD

**Discussion by joconnor-ecaa** [developmentseed/tipg#128] [Q&A] [answered]
Hi all, thank you for the great work.

Do you have plans to publish the docker images specified in this repo? We're currently building them ourselves but it would be nice to point at the official version.

**Comment by vincentsarago** [comment, marked as answer]
Hi @joconnor-ecaa 

We do publish an image https://github.com/developmentseed/tipg/pkgs/container/tipg 

I think we just forgot to add a note in the documentation 😅

**Reply by vincentsarago** [reply]
updated the docs https://github.com/developmentseed/tipg#docker-images
